nilearn/tests/test_niimg_conversions.py

```diff
@@ -164,30 +164,20 @@ def test_concat_niimgs():
     img1c = Nifti1Image(np.ones(shape3), affine)
 
     # check basic concatenation with equal shape/affine
-    concatenated = _utils.concat_niimgs((img1, img3, img1),
-                                        accept_4d=False)
-    concatenate_true = np.ones(shape + (3,))
+    concatenated = _utils.concat_niimgs((img1, img3, img1))
 
-    # Smoke-test the accept_4d
-    assert_raises_regex(ValueError, 'image',
+    assert_raises_regex(TypeError, 'image',
```

Comment:
could we have a more explicit message checking that 'image' while you are at it ?